backend/users/tests/test_registration.py

```diff
@@ -46,3 +52,44 @@ def test_register_verification(client, db, django_user_model):
     assert allauth_email.email == creds.get('email')
     assert allauth_email.verified is True
     assert allauth_email.primary is True
+
+
+def test_register_throttling(client, throttle_settings):
```

Comment:
> The throttle rate from the fixture is not actually applied, although its value is read correctly in the test function

Perhaps an obvious suggestion, but did you try this?

```suggestion
def test_register_throttling(throttle_settings, client):
```

Reply:
I've now tried this, but the result is the same.